WcaOnRails/db/structure.sql

```diff
@@ -1241,9 +1241,11 @@ INSERT INTO `schema_migrations` (version) VALUES
 ('20170823170616'),
 ('20170830140540'),
 ('20170831170616'),
+('20170930231828'),
```

Comment:
Is this a phanton migration?

Reply:
Yeah, it's from my delegates table work.